deploy/docs/Upgrading_with_Helm.md

```diff
@@ -0,0 +1,78 @@
+|   `--no-hooks`   	| Disable pre/post upgrade hooks                                                                                                                           	|
+| `--reset-values` 	| When upgrading, reset the values to the ones built into the chart                                                                                        	|
+| `--reuse-values` 	| When upgrading, reuse the last releases values and merge in any overrides from the command line via `--set` and `-f`                                        	|
+|  `--set` stringArray	| Set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)                                          	|
+|   `--set-file` stringArray  	| Set values from respective files specified via the command line (can specify multiple or separate values with commas: key1=path1,key2=path2) 	|
+|  `--set-string` stringArray  	|  Set STRING values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)                             	|
+|    `--version`   	| Specify the exact chart version to use. If this is not specified, the latest version is used                                                             	|
+
+
+To edit or append to the existing customized values, add the `–reuse-values` flag, otherwise any existing customized values are ignored.
+
+If no chart value arguments are provided on the command line, any existing customized values are carried forward, i.e. the `--reuse-values` flag will be used by default. If you want to revert to just the values provided in the chart, use the `-–reset-values` flag.
+
+And if there exists `--set/--set-file/--set-string/--values/-f` , the `--reset-values` flag will be used by default.
+
+### Example using the `--set` flag in the upgrade command:
+
+```bash
+helm upgrade collection sumologic/sumologic --set key1=val1, key2=val2
+```
+
+### Example using the `-f` flag in the upgrade command:
+
+You can specify any of the chart value flags multiple times. The priority will be given to the last (right-most) value specified. For example, if both `myvalues.yaml` and `override.yaml` contained a key called ‘Test’, the value set in `override.yaml` would take precedence:  
+
+```bash
+$ helm upgrade collection sumologic/sumologic --reuse-values -f myvalues.yaml -f override.yaml
+```
+
+In the above case, the `collection` release is upgraded with the same chart, but with a new  key-value pair in the YAML file:
+```bash
+Test: true
+```
+  
+We can use helm `get values` to see whether that new setting took effect.
+```bash
+$ helm get values collection
+Test: true
+```
+
+The helm get command is a useful tool for looking at a release in the cluster. And as we can see above, it shows that our new values from `override.yaml` were deployed to the cluster.
```

Comment:
Can we also wrap "helm get" as an in-line command, like "The `helm get` command is..."?